DigitalLearningSolutions.Data.Tests/Extensions/EnumerableExtensionsTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Extensions
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Extensions;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class EnumerableExtensionsTests
+    {
+        [Test]
+        public void WhereNotNull_filters_items_correctly()
+        {
+            // Given
+            var list = new List<string?> { "1", "2", null, "3", null, "4" };
+            var expectedResult = new List<string?> { "1", "2", "3", "4" };

```

Comment:
Does the nullability of the type make a difference? I would expect a `List<string>`.
Also, I've a bit of a preference for inlining expected output in the `Then` section - that's where I'd habitually look for the expected output, because often most of the literals in the `Given` section are input data or setting up mocks.

Reply:
Ah whoops, this was a copy and paste error, it returns `IEnumerable<T>` from input `IEnumerable<T?>`